examples/reference-example/example-config.hcl

```diff
@@ -19,7 +19,7 @@ task {
   version = "1.0.0"
   providers = ["myprovider"]
   services = ["web", "api"]
-  variable_files = ["example.module.tfvars"]
+  variable_files = ["/path/to/example.module.tfvars"]
```

Comment:
Updated this to show that it currently requires an absolute path (see issue https://github.com/hashicorp/consul-terraform-sync/issues/279).